From 4e364cf295f101443bff99e0fa3325260580ae04 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Tue, 1 Jan 2013 11:39:30 +0100
Subject: [PATCH 3/4] build the docs

---
 debian/rules |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/debian/rules b/debian/rules
index e086a54..038ee24 100755
--- a/debian/rules
+++ b/debian/rules
@@ -36,3 +36,8 @@ override_dh_auto_configure:
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		$(CMAKE_FLAGS)
 
+override_dh_auto_build:
+	if [ -f "/opt/ros/groovy/setup.sh" ]; then . "/opt/ros/groovy/setup.sh"; fi && \
+	dh_auto_build && make docs
+
+
-- 
1.7.9.5

