From 02ce16a9b8e0d49964c73af87f508b59ddff1c5d Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Fri, 26 Oct 2012 12:26:15 -0700
Subject: [PATCH 6/6] fix the call to catkin

---
 CMakeLists.txt |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2cc2457..5a2ec4f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,13 +58,14 @@ if(DEFINED CMAKE_BUILD_TYPE AND CMAKE_VERSION VERSION_GREATER "2.8")
   set_property( CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS ${CMAKE_CONFIGURATION_TYPES} )
 endif()
 
+project(opencv2)
+find_package(catkin)
+catkin_package()
+
 project(OpenCV CXX C)
 
 include(cmake/OpenCVUtils.cmake REQUIRED)
 
-find_package(catkin)
-catkin_package()
-
 # ----------------------------------------------------------------------------
 # Break in case of popular CMake configuration mistakes
 # ----------------------------------------------------------------------------
-- 
1.7.10.4

